### DISCUSSION TITLE
Once the commit is done, my Github job statuses are reset

### DISCUSSION THREAD

**Discussion by muratkeremozcan** [stefanzweifel/git-auto-commit-action#310] [Help]
I want to keep my job statuses, so that required jobs can stay there and branch protection works. 
Alas, the action, after it runs and commits the files, resets the github job statuses. What can we do to make this work right?

**Comment by stefanzweifel** [comment]
Hi

This is _by design_ and a limitation of how GitHub Actions works.
The statuses are not _reset_ per se, but they don't run on the newly created commit.

You can find more information about this [here](https://github.com/stefanzweifel/git-auto-commit-action?tab=readme-ov-file#commits-made-by-this-action-do-not-trigger-new-workflow-runs).

You can solve this by creating a personal access token and add it to the `actions/checkout`-step.

## Related

- https://github.com/stefanzweifel/git-auto-commit-action/discussions/267#discussioncomment-4286556
